siliconcompiler/tools/builtin/verify.py

```diff
@@ -0,0 +1,72 @@
+from siliconcompiler.tools.builtin import _common
+from siliconcompiler.schema import Schema
+
+import re
+
+def setup(chip):
+    '''
+    Tests an assertion on a list of input tasks.
+
+    The provided steplist is verified to ensure that all assertions
```

Comment:
"provided steplist" -> "input step"? Original wording makes it seem like it operates on entire steplist, but it only takes a single input.